shipyard/src/main/java/org/jclouds/shipyard/handlers/ShipyardErrorHandler.java

```diff
@@ -90,7 +94,7 @@ private String parseMessage(HttpResponse response) {
       try {
          return Strings2.toStringAndClose(response.getPayload().openStream());         
       } catch (IOException e) {
-         throw Throwables.propagate(e);
+         return Throwables.getStackTraceAsString(e);
```

Comment:
Fair enough. I guess my thinking was that the exception turned-into-string would get put inside the exception anyway.